regexes/client/browsers.yml

```diff
@@ -319,7 +319,7 @@
     default: 'Blink'
 
 # Chromium GOST (https://github.com/deemru/chromium-gost)
-- regex: 'Chrome/(\d+[\.\d]+).+\(Chromium GOST\)'
+- regex: 'Chrome/(\d+\.[\.\d]+)[^\.\d].*\(Chromium GOST\)'
```

Comment:
I would remove the condition after the version `[^\.\d]` since this is any character, or rather it can be replaced with a `space`.

```yml
- regex: 'Chrome/(\d+\.[.\d]+) .*\(Chromium GOST\)'
```

Reply:
OK I've changed the ones I can to a space.
The others I wasn't sure about so left with `[^\.\d]` to avoid `[\.\d]+` being directly next to `.*` (quadratic worst-case).